change-notes/2022-08-24-missing-check-scanf.md

```diff
@@ -0,0 +1,4 @@
+---
+category: newQuery
+---
+* Added a new query, `cpp/missing-check-scanf`, to detect `scanf` output variables that are used without a proper return-value check to see that they were actually written.
```

Comment:
We should add that this query was initially submitted as an experimental query. You can see the phrasing I used for this [here](https://github.com/github/codeql/pull/5930).

Reply:
The change note is in the wrong place by the way, which is why the CI is still complaining. Should be in `cpp/ql/src/change-notes`. I get why you added it where you added it. It's probably the most obvious place for it, and holds change notes for pretty old versions 😄

Reply:
Ah, ok, I was wondering about that. Because the check-change-note GitHub action seemed buggy to me. It does `gh api 'repos/github/codeql/pulls/10163/files' --paginate --jq 'any(.[].filename ; test("/change-notes/.*[.]md$"))'`, which returns false; but if only it did `test("change-notes/.*[.]md$")`, it would have returned true. But now I see it's a matter of moving the change note to a subdirectory.